src/components/core/geoChart/GeoChartLegendRenderer.tsx

```diff
@@ -1,12 +1,69 @@
-// (C) 2007-2019 GoodData Corporation
+// (C) 2020 GoodData Corporation
 import * as React from "react";
+import without = require("lodash/without");
+import * as classNames from "classnames";
+import { Execution } from "@gooddata/typings";
+import { stringToFloat } from "../../../helpers/utils";
+import { IntlWrapper } from "../../core/base/IntlWrapper";
+import { IntlTranslationsProvider, ITranslationsComponentProps } from "../../core/base/TranslationsProvider";
+import PushpinSizeLegend from "./legends/PushpinSizeLegend";
+import { getGeoData, calculateAverage, getFormatFromExecutionResponse } from "../../../helpers/geoChart";
+import { IGeoConfig, IGeoData } from "../../../interfaces/GeoChart";
+import { TOP } from "../../visualizations/chart/legend/PositionTypes";
+import { isTwoDimensionsData } from "../../../helpers/executionResultHelper";
 
-export interface IChartLegendProps {
-    domProps: any;
+export interface IGeoChartLegendRendererProps {
+    config: IGeoConfig;
+    execution: Execution.IExecutionResponses;
+    locale: string;
+    position?: string;
 }
+export default function GeoChartLegendRenderer(props: IGeoChartLegendRendererProps): JSX.Element {
+    const {
+        execution: { executionResult, executionResponse },
+        config: { mdObject: { buckets = [] } = {} },
+        locale,
+        position = TOP,
+    } = props;
+    const geoData: IGeoData = getGeoData(buckets, executionResponse.dimensions);
+    const { data } = executionResult;
+    const classes = ["geo-legend", "s-geo-legend", `position-${position}`];
```

Comment:
it could be just, and we don't need to use `className` since there is no condition
```
const classes = `geo-legend, s-geo-legend, position-${position}`;
```

Reply:
fixed